### PR TITLE
[Inductor Perf CI] Enable perf CI smoke test

### PR DESCRIPTION
**Pull request by weiwangmeta**
This tries to detect perf regression e.g. https://github.com/pytorch/pytorch/pull/91316#issuecomment-1370370885

### CODE REVIEW

**Comment by weiwangmeta** [comment]
Not ready to merge yet, close for now.

**Comment by weiwangmeta** [comment]
Would re-opening this get those six subsequent commits?

**Review by seemethere — APPROVED** [review]

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -302,9 +302,15 @@ test_inductor_benchmark_perf() {
   # Check training with --amp
   # Not checking accuracy for perf test for now
   # shellcheck disable=SC2086
-  python benchmarks/dynamo/$1.py --ci --training --performance --disable-cudagraphs\
-    --device cuda --inductor --amp $PARTITION_FLAGS  --output "$TEST_REPORTS_DIR"/inductor_training_$1.csv
+  if [[ "$1" == *smoketest* ]]; then
+    python benchmarks/dynamo/torchbench.py --performance --backend inductor --float16 --training \
+      --batch-size-file "$(realpath benchmarks/dynamo/torchbench_models_list.txt)" --only hf_Bert
```
Comment (desertfire) [inline .jenkins/pytorch/test.sh:307]: Please specify `--output` here so the generated csv file is uploaded.

**Comment by desertfire** [comment]
cc @ngimel

**Review by desertfire — CHANGES_REQUESTED** [review]
Please address my comment about `--output`

File: .jenkins/pytorch/test.sh [hunk 2]
```diff
@@ -337,6 +343,10 @@ test_inductor_torchbench_perf() {
   PYTHONPATH=$(pwd)/torchbench test_inductor_benchmark_perf torchbench
 }
 
+test_inductor_torchbench_smoketest_perf(){
+  PYTHONPATH=$(pwd)/torchbench test_inductor_benchmark_perf smoketest
```
[the remark this answers is not in the document]
Comment (weiwangmeta) [reply .jenkins/pytorch/test.sh:348]: Yes, I will do a BE fix in future.

File: .jenkins/pytorch/test.sh (hunk 1 above)
Comment (weiwangmeta) [reply .jenkins/pytorch/test.sh]: Yes, done.

**Review by desertfire — APPROVED** [review]
Let's monitor the perf number for several PRs, and add perf number guarding if the number turns out to be stable.

**Comment by weiwangmeta** [comment]
Sounds good. I will start merging this and add number guarding after we are able to collect numbers from commits pushed. 

@pytorchbot merge

**Pull request closed without merging**